doc/args_github_handler.rst

```diff
@@ -3,7 +3,45 @@
 GitHub Handler command line
 ---------------------------
 
-.. Insert some description here
+GitHub Handler is Universum mode that serves as GitHub Application, helping to perform and report checks on
+new commits to a repository. In this mode Universum can create new check runs on GitHub and trigger
+builds to perform these checks, parsing accepted payload for required params and passing them to usual
+Universum runs in these builds. The following picture represents the event sequence leading to check result
+for a commit displayed on GitHub:
+
+*(picture here)*
+
+On this picture numbers depict the following events:
+
```

Comment:
Please either add a short description of what has to be setup to make the described workflow to work, or change the workflow description into the setup tutorial.